pkg/admin/facade/rolesgroups.go

```diff
@@ -80,9 +82,13 @@ func (f *RolesGroupsFacade) ListRoles(options *rolesgroups.ListRolesOptions, pag
 		return nil, nil, err
 	}
 
+	count, err := f.RolesGroupsQueries.CountRoles()
+	if err != nil {
+		return nil, nil, err
+	}
+
```

Comment:
ListRoles supports `SearchKeyword`, so CountRoles should also support it?

Reply:
It's mainly for counting the total number. Do you think it's better to add it?

Reply:
@Ngkaokis I think pagination will fail if the total count doesn't respect the search keyword?

Reply:
If we stick we the behaviour as users list, searching should not show pagination and would query all results (although we limit the search result to 10 in users list for performance issue. But I think we don't have issue with roles and groups query)

Reply:
Oh I see. Lets keep it return null if search keyword exist first to be consistent with user query. And lets confirm it with @fungc-io later.